src/test/java/org/broadinstitute/hellbender/utils/read/ReadUtilsUnitTest.java

```diff
@@ -413,24 +413,31 @@ public void testSetReadsAsSupplemental() {
         final SAMFileHeader header = ArtificialReadUtils.createArtificialSamHeader();
         GATKRead primarySup = ArtificialReadUtils.createArtificialRead(header, "read1", 1, 10000, new byte[] {}, new byte[] {}, "4M");
         primarySup.setMappingQuality(100);
+        primarySup.setAttribute("NM",20);
```

Comment:
This is great because we are expecting virtually all of our data to have the NM tag. If you need a bam that has those tags for the integration tests I think the K-562 should have it.